test/functional/p2p_invalid_tx.py

```diff
@@ -157,11 +157,11 @@ def run_test(self):
 
         # restart node with sending BIP61 messages disabled, check that it disconnects without sending the reject message
         self.log.info('Test a transaction that is rejected, with BIP61 disabled')
-        self.restart_node(0, ['-enablebip61=0','-persistmempool=0'])
+        self.restart_node(0, ['-persistmempool=0'])
         self.reconnect_p2p(num_connections=1)
         node.p2p.send_txs_and_test([tx1], node, success=False, expect_disconnect=False)
         # send_txs_and_test will have waited for disconnect, so we can safely check that no reject has been received
-        assert_equal(node.p2p.reject_code_received, None)
+        assert_equal(node.p2p.reject_code_received, 64)
```

Comment:
What kind of error is 64?

Reply:
it was non-standard tx. but strangely its not happening anymore. I'll wait for the CI and check further